packages/light-react/src/Send/TxForm/TxForm.js

```diff
@@ -0,0 +1,148 @@
+// Copyright 2015-2018 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
+//
+// SPDX-License-Identifier: MIT
+
+import React, { Component } from 'react';
+import { defaultAccount$ } from '@parity/light.js';
+import { fromWei, toWei } from '@parity/api/lib/util/wei'; // TODO @parity/api's wei util should have Gwei as an unit
+import { inject, observer } from 'mobx-react';
+import light from 'light-hoc';
+import { Link } from 'react-router-dom';
+
+import TokenBalance from '../TokenBalance';
+
+const MAX_GAS_PRICE = 40; // In Gwei
+const MIN_GAS_PRICE = 3; // Safelow gas price from GasStation, in Gwei
```

Comment:
At some point, it would be great to have these pull in via API. (Probably not for v1)

Reply:
You mean from an external source like eth gas station?